### DISCUSSION TITLE
一些功能的请教

### DISCUSSION THREAD

**Discussion by ilovedumplings** [alibaba/higress#1472] [Q&A]
## Why do you need it?
我们在实际使用当中有一些场景,可能需要一些插件支持,想请教下,我们是否支持这些功能,不支持的功能,我们自己尝试开发
1.AI缓存对于意图的识别,就是对于时间相关性问题的缓存,这些其实是不应该缓存的。 
johnlanni 大哥给了回答,说是应该不支持
2.我们现在具备了限流功能,是否具备熔断机制?
higress好像有集成sentinel我理解这也是可以的 
3.在AI配额中,我们是否支持针对请求和响应的配额进行分开配置
4.还有就会死监控的扩展,我看监控这块本身使用的Prometheus,待我研究研究,再来提问
@CH3CHO 

## How could it be?
A clear and concise description of what you want to happen. You can explain more about input of the feature, and output of it.

**Comment by ilovedumplings** [comment]
@johnlanni 请问
![image](https://github.com/user-attachments/assets/89048888-7024-4cd2-9cf1-baa9e356fe9e)
这个功能有出嘛?

**Comment by CH3CHO** [comment]
> @johnlanni 请问 ![image](https://private-user-images.githubusercontent.com/19704134/382204543-89048888-7024-4cd2-9cf1-baa9e356fe9e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA2ODM2MjIsIm5iZiI6MTczMDY4MzMyMiwicGF0aCI6Ii8xOTcwNDEzNC8zODIyMDQ1NDMtODkwNDg4ODgtNzAyNC00Y2QyLTljZjEtYmFhOWUzNTZmZTllLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA0VDAxMjIwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFlMzMyYzExN2JkMjZlOGIwMDc2YmFhMzFkMjBiZWYyOWJiNmExZjFlMjg4NzdiYjAwODZjOTBmYWZhMmZhNTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.l8zqOsMDNF2BNX1O15Cwi3Zb8jhYRlHWuAM9d8lawRM) 这个功能有出嘛?

我记得这个前两天群里有人问，是还没有开源的。

**Comment by CH3CHO** [comment]
> 1.AI缓存对于意图的识别,就是对于时间相关性问题的缓存,这些其实是不应该缓存的。 johnlanni 大哥给了回答,说是应该不支持 

见 https://github.com/alibaba/higress/issues/1459

> 2.我们现在具备了限流功能,是否具备熔断机制? higress好像有集成sentinel我理解这也是可以的

单说熔断的话，Envoy 本身就具备一定这样的机制，比如说 [circuit breaking](https://www.envoyproxy.io/docs/envoy/v1.27.0/intro/arch_overview/upstream/circuit_breaking)、[outlier detection](https://www.envoyproxy.io/docs/envoy/v1.27.0/intro/arch_overview/upstream/outlier) 等。

> 3.在AI配额中,我们是否支持针对请求和响应的配额进行分开配置

现在是不支持的，但可以单独提个 issue 我们讨论一下。为什么会有这方面的需求呢？
